api/tests/serializers_tests.py

```diff
@@ -48,7 +57,7 @@ def test_premium_can_set_block_list_emails(self):
 
         url = reverse("relayaddress-detail", args=[premium_alias.id])
         data = {"block_list_emails": True}
-        free_token = Token.objects.get(user=premium_user)
+        free_token = self._get_token_for_user(premium_user)
```

Comment:
nit: should be `premium_token` for a `premium_user`. Or, could just use `token`.